independent-publisher-connectors/Google Chart QR Code/apiDefinition.swagger.json

```diff
@@ -0,0 +1,120 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Google Chart QR Code Generator",
+    "description": "This Connector is build arount the Google Chart API to generate QR codes to be used in Power Platform applications",
+    "version": "1.0",
+    "contact": {
+      "name": "Iona Varga",
+      "email": "ionavarga@outlook.com"
+    }
+  },
+  "host": "chart.googleapis.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "x-ms-connector-metadata": [
+  {
+      "propertyName": "Website",
+      "propertyValue": "https://chart.googleapis.com/"
+  },
+  {
+    "propertyName": "Privacy policy",
+    "propertyValue": "https://developers.google.com/chart/interactive/docs/security_privacy"
+  },
+  {
+    "propertyName": "Categories",
+    "propertyValue": "Data"
+  }
+  ],
+  "paths": {
+    "/chart": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "$content-type": {
+                  "type": "string",
+                  "description": "$content-type"
+                },
+                "$content": {
+                  "type": "string",
+                  "description": "$content"
+                }
+              }
+            }
+          }
+        },
+        "operationId": "GenerateQR",
+        "summary": "Generate a QR code based on a set of inputs",
+        "description": "By providing a width and heigth, data, encoding and correction level, the Google Chart API can generate a QR code.",
```

Comment:
@Ionavoss All files should be free of grammatical and spelling errors. in the description you have "By providing a width and heigth, data...", maybe you meant to say "By providing a width and **height**, data..."? Can you please fix?